src/styles/main.css

```
@@ -0,0 +1,279 @@
+  background-color: var(--color-bluegray-100);
+}
+
+#wrap,
+section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+main {
+}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)